### PR TITLE
Add a feature for handling scheduled_time in ahi_hsd reader.

### PR DESCRIPTION
**Pull request by tsukada-cs**
Added min and sec information to `scheduled_time` in ahi_hsd reader.

The `scheduled_time` had an accuracy of 10 min, but the Japan region (JPee) and Region 3 (R3ff) are observed at a frequency of 150 sec. Regions 4 (R4gg) and 5 (R5ii) of non-public data are observed at a frequency of 30 sec.
Detailed information on the Himawari Standard Data is found [here](http://www.data.jma.go.jp/mscweb/en/himawari89/space_segment/hsd_sample/HS_D_users_guide_en_v13.pdf).
With this PR, `scheduled_time` is calculated correctly for those observations.

 - [x] Tests added and test suite added to parent suite
 - [x] Tests passed
 - [x] Passes ``flake8 satpy``
 - [x] Add your name to `AUTHORS.md` if not there already

### CODE REVIEW

File: AUTHORS.md [hunk 1]
```diff
@@ -62,3 +62,4 @@ The following people have made contributions to this project:
 - [praerien (praerien)](https://github.com/praerien)
 - [Xin Zhang (zxdawn)](https://github.com/zxdawn)
 - [Yufei Zhu (yufeizhu600)](https://github.com/yufeizhu600)
+- [Taiga Tsukada (tsukada-cs)](https://github.com/tsukada-cs)
```
Comment (mraspaud) [inline AUTHORS.md:65]: Please insert your name in alphabetical order in the list.

Comment (tsukada-cs) [reply AUTHORS.md]: I wrote the test. However, I am not used to writing tests, so please check/advice me.

Comment (mraspaud) [reply AUTHORS.md]: This looks good. The test is run and passes.

**Comment by mraspaud** [comment]
This is good to merge for me. If no one else says otherwise, I'll merge tomorrow. @sfinkens @simonrp84

File: AUTHORS.md (hunk 1 above)
Comment (tsukada-cs) [reply AUTHORS.md]: Thank you for your kind response.

**Comment by simonrp84** [comment]
Yes, this looks good to me too, thanks for adding it!

Out of interest, @tsukada-cs, is it possible to order / request the 30 second AHI data somewhere? I would be really interested in using that for my research!

**Comment by tsukada-cs** [comment]
> Yes, this looks good to me too, thanks for adding it!
> 
> Out of interest, @tsukada-cs, is it possible to order / request the 30 second AHI data somewhere? I would be really interested in using that for my research!

Our research group has been able to obtain 30-second data in collaboration with the [Japan Meteorological Research Institute (MRI)](https://www.mri-jma.go.jp/index_en.html).
So, I think you should contact the MRI first.

**Comment by sfinkens** [comment]
Very nice, thanks for your contribution @tsukada-cs !

**Pull request merged**